server/docker-base/Dockerfile

```diff
@@ -349,13 +349,16 @@ RUN \
 # Switch SDK
 #
 
-ENV NINTENDO_SDK_ROOT ${PLATFORMSDK_DIR}/nx
-ENV SWITCH_SDK_FILENAME nx64-sdk-10.4.1.tar.gz
+ENV NINTENDO_SDK_ROOT_13_3_0 ${PLATFORMSDK_DIR}/nx-13.3.0
```

Comment:
It feels safer to keep version numbers in the variable, so we don't accidentally update for users that want to stay on a version for a while